Sources/_StringProcessing/Unicode/NFC.swift

```diff
@@ -9,8 +9,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-@_spi(_Unicode)
-import Swift
+@_spi(_Unicode) import Swift
```

Comment:
I'd prefer the attribute to appear on a newline above the import.

Reply:
Ok. How do you see this style mixing with multiple imports in the same file? To me this looked rather unusual and made it hard to scan the block of imports. I could also add a newline before `internal import _RegexParser` but that too looks unusual.

Reply:
My personal preference for attributes is always above the declaration it's attributing to. In the standard library for functions we have:
```swift
@available(SwiftStdlib 6.0, *)
func something() {
}
```
right, and for property wrappers my preference is also above the property declaration:
```swift
struct A {
  @propertyWrapper
  let b: Int

  let c: String
}
```
I think the newline before the 2nd property here makes sense to me and would similarly apply it to imports:
```swift
@attribute
import Swift

import A
import B
```
But again, this is just personal preference. I don't think we have a defined style yet for this because we just simply don't write imports with attributes often at all.

Reply:
I generally agree that attributes belong above declarations, especially when the list of attributes is likely to grow and line lengths can become an issue. For whatever reason, though, vibes-wise I don't think of `imports` as following the same rules as other declarations because it's so common for there to be an uninterrupted block of them at the top of a file. `@_spi` on a separate line really sticks out like a sore thumb to me, never seen it outside of this codebase. I'll follow your recommendation here, though, as I agree it seems to be the most consistent rule.